### PR TITLE
Storybook update : dropdown component

### PR DESCRIPTION
**Pull request by Marine-Berthier**
Stories for the dropdown component written in v6 format

### CODE REVIEW

**Comment by ralucacril** [comment]
you might need to merge the original branch back into yours since you have some conflicts. nice work! 🎉  👏

**Review by ralucacril — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "tailwindcss": "^1.4.0"
   },
   "dependencies": {
-    "bindings": "^1.5.0",
+    "@storybook/addon-knobs": "^6.1.6",
```
Comment (masone) [inline package.json:33]: seems like you were not on the latest version when you branched off the branch `update-storybook`? you can try 
```
git checkout update-storybook
git pull
git checkout dropdown
git rebase update-storybook
```

**Comment by Marine-Berthier** [comment]
I merged the original branch to `dropdown`, it seems there's no more conflict. Should I merge right now ?

File: src/stories/dropdown.stories.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import React from "react"
+import Dropdown from "../components/dropdown/index"
+import { action } from "@storybook/addon-actions"
+import { object, number, boolean, text } from "@storybook/addon-knobs"
```
Comment (masone) [inline src/stories/dropdown.stories.js:4]: I understood we wanted to go without knobs? did you find them to be valuable after all and we actually want them in?

Comment (ralucacril) [reply src/stories/dropdown.stories.js:4]: i think @Averethel said he sees values in knobs

Comment (Marine-Berthier) [reply src/stories/dropdown.stories.js:4]: For this particular component I could not find another way to make it work, but I can try harder :blush: However there is no Knobs pannel created, everything is under Controls pannel.

Comment (Marine-Berthier) [reply src/stories/dropdown.stories.js:4]: Actually, everything works perfectly fine without them 🤦 I was looking for something complicated when actually it was very simple ! Should I push the changes (stories without knobs) ?

Comment (Averethel) [reply src/stories/dropdown.stories.js:4]: @ralucacril I did, but @Marine-Berthier explained that there's now a different way to play around with component. Let's roll without it for now and if we see we need them - we act.

Comment (ralucacril) [reply src/stories/dropdown.stories.js:4]: a ok. my bad. then @Marine-Berthier roll them without the knobs if is working

Comment (Marine-Berthier) [reply src/stories/dropdown.stories.js:4]: I realised with this component that I didn't really understand the purpose of knobs. I thought that knobs could be replaced by args as soon as it is now possible to play with args the same way you can play with knobs. Actually it is still 2 different things : knobs could still be used and won't appear as args in the code. I might be wrong though

**Pull request merged**